packages/vue3-jest/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "vue3-jest",
+  "version": "26.0.0-alpha.10",
+  "description": "Jest Vue transform",
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "keywords": [
+    "jest",
+    "vue",
+    "jest vue",
+    "jest vue transform",
+    "jest vue preprocessor",
+    "vue jest",
+    "vue jest",
+    "vue jest transform",
+    "vue jest preprocessor"
+  ],
+  "scripts": {
+    "release": "semantic-release",
+    "test": "echo \"There are no unit tests yet. Run e2e/3.x/ tests instead.\""
+  },
+  "author": "Edd Yerburgh",
+  "license": "MIT",
+  "devDependencies": {
+    "@babel/core": "^7.9.0",
+    "@vue/compiler-sfc": "^3.0.3",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "^26.0.0",
+    "conventional-changelog": "^1.1.5",
+    "jest": "^26.0.0",
+    "semantic-release": "^15.13.2",
+    "ts-jest": "^26.4.4",
+    "typescript": "^4.1.2",
+    "vue": "^3.0.3"
+  },
+  "peerDependencies": {
+    "@babel/core": "7.x",
+    "babel-jest": ">= 24 < 27",
+    "jest": ">= 24 < 27 ",
```

Comment:
I think we should only match Jest 26 now.

Reply:
Changed to `26.x`.